manual/additional_tools.tex

```diff
@@ -10,28 +10,41 @@ \section{Initialization Tools}
 
 \section{Post-Processing}
   \subsection{qmca}
-    qmca is a versatile tool to analyze and plot the raw data from QMCPACK *.scalar.dat files.
+    \texttt{qmca} is a versatile tool to analyze and plot the raw data from QMCPACK *.scalar.dat files.
     It is a python executable and part of the Nexus suite of tools.  It can be found in 
-    \texttt{trunk/nexus/executables}. 
-  \subsection{qmcfit}
+    \texttt{trunk/nexus/executables}. For more detail, see section \ref{sec:qmca}.
+  \subsection{qfit}
+    \texttt{qfit} is a curve fitting tool to obtain statistical error bars on fitted parameters.
+    It is useful for DMC timestep extrapolation.  For more details, see section \ref{sec:qfit}.
   \subsection{qmcfinitesize}
-    qmcfinitesize is a utility to compute many-body finite-size corrections to the energy.  It
+    \texttt{qmcfinitesize} is a utility to compute many-body finite-size corrections to the energy.  It
     is a C++ executable that is built alongside the qmcpack executable.  It can be found in 
     \texttt{trunk/build/bin}
-  \subsection{trace-density}
 
 \section{Converters} 
 \input{convert4qmc}    
-  \subsection{wfconvert}
-    wfconvert allows the conversion of wavefunction from another code into a format supported by QMCPACK.
-    Currently most its important features are available within QMCPACK and there is no need to use it anymore.
-    Converting wavefunctions from LAPW, nonuniform b-splines, wannier functions has not been transferred.
   \subsection{pw2qmcpack.x}
     pw2qmcpack.x is an executable that converts PWSCF wavefunctions to QMCPACK readable 
     HDF5 format.  This utility is built alongside the Quantum Espresso post-processing utilities.
     This utility is written in Fortran90, and is distributed as a patch of the Quantum Espresso 
     source code.  The patch, as well as automated QE download and patch scripts, can be found in 
     \texttt{trunk/external\_codes/quantum\_espresso}.
+
+pw2qmcpack used to be used in serial without problems in small systems and becomes a severe bottleneck for a large system.
+Due to the large energy cutoff in the calculation with pw.x required by the pseudopotentials and the increasing sizes of systems we are interested in, one limitation of QE can be easily reached
+that \texttt{wf\_collect=.true.} results in problems of writing and loading correct plane wave coefficients on disks by pw.x because of the 32bit integer issue. Thus, pw2qmcpack.x fails to convert the orbitals for QMCPACK.
+
+Since Quantum ESPRESSO 5.3.0, the new converter has been fully parallelized to overcome that limitation completely.
+By removing \texttt{wf\_collect=.true.} (by default \texttt{wf\_collect=.false.}), pw.x doesn't collect the whole wave function into individual files for each k point but write one small file for each processor instead.
```

Comment:
QE changed default to true since 6.2 (maybe 6.1)

Reply:
Let me fix the context. I need to update tests also.

Reply:
@ye-luo Please do a later PR that updates all these docs after the QE 6.3 converter is merged.